lib/views/on_boarding_view/welcome_screen_view.dart

```diff
@@ -0,0 +1,136 @@
+        child: Padding(
+          padding: AppPadding.screenPadding,
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: <Widget>[
+              const Spacer(),
+              Image.asset(
+                'assets/images/streamicon.png',
+                width: 150.0,
+                height: 150.0,
+              ),
+              const SizedBox(height: 24),
+              _buildWelcomeText(),
+              const SizedBox(height: 8),
+              _buildOverviewText(),
+              const SizedBox(height: 48),
+              const Spacer(),
+              _buildLoginButton(context, ref),
+              const SizedBox(height: 12),
+              _buildContinueWithoutButton(),
+              const SizedBox(height: 12),
+              _buildInternalAccountLink(context),
+              const Spacer(flex: 2),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildWelcomeText() {
+    return const Text(
+      'Welcome to Gocast',
+      textAlign: TextAlign.center,
+      style: TextStyle(
+        fontSize: 24,
+        fontWeight: FontWeight.bold,
+        color: Colors.black,
+      ),
+    );
+  }
+
+  Widget _buildOverviewText() {
+    return const Text(
+      "Your Lectures on the Go",
+      textAlign: TextAlign.center,
+      style: TextStyle(fontSize: 16, color: Colors.black54),
+    );
+  }
+
+  Widget _buildLoginButton(BuildContext context, WidgetRef ref) {
+    final viewModel = ref.watch(userViewModelProvider.notifier);
+    return ElevatedButton(
+      style: ElevatedButton.styleFrom(
+        foregroundColor: Colors.white,
+        backgroundColor: Colors.blue[900],
+        padding: const EdgeInsets.symmetric(
+          vertical: AppSizes.buttonVerticalPadding,
+        ),
+        shape:
+            RoundedRectangleBorder(borderRadius: BorderRadius.circular(15.0)),
+      ),
+      child: viewModel.current.value.isLoading
+          ? const SizedBox(
+              width: 20.0,
+              height: 20.0,
+              child: CircularProgressIndicator(
+                valueColor: AlwaysStoppedAnimation<Color>(Colors.white),
+              ),
+            )
+          : const Text('TUM Login', style: TextStyle(fontSize: 18)),
+      onPressed: () => handleSSOLogin(context, ref),
+    );
+  }
+
+  Widget _buildContinueWithoutButton() {
+    return OutlinedButton(
+      style: OutlinedButton.styleFrom(
+        side: BorderSide(color: Colors.blue[900] ?? Colors.blue),
+        foregroundColor: Colors.blue[900],
+        padding: const EdgeInsets.symmetric(
+          vertical: AppSizes.buttonVerticalPadding,
+        ),
+        shape:
+            RoundedRectangleBorder(borderRadius: BorderRadius.circular(15.0)),
+      ),
+      child: const Text('Continue without', style: TextStyle(fontSize: 18)),
+      onPressed: () {
+        //TODO: Continue without login action
```

Comment:
Last suggestion: is it possible to implement this right now? Otherwise the TestFlight version is not really usable beyond the WelcomeScreen

Reply:
Should be no problem - I'll have a look at it now

Reply:
Sounds good as a temporary solution. Once the API is deployed things get much clearer I suppose